lib/commands/release/index.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import { Release } from 'balena-sdk';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, getVisuals, stripIndent } from '../../utils/lazy';
+
+interface FlagsDef {
+	help: void;
+	composition?: boolean;
+}
+
+interface ArgsDef {
+	commit?: string;
+}
+
+export default class ReleaseCmd extends Command {
+	public static description = stripIndent`
+		Get info for a release.
+`;
+	public static examples = ['$ balena release commit'];
+
+	public static usage = 'release <commit>';
+
+	public static flags: flags.Input<FlagsDef> = {
+		help: cf.help,
+		composition: flags.boolean({
+			default: false,
+			char: 'c',
+			description: 'Return the release composition',
+		}),
+	};
+
+	public static args = [
+		{
+			name: 'commit',
+			description: 'the commit of the release to get information',
```

Comment:
Maybe we could also document using release IDs, since commits are not even unique in the same fleet.
(The commit is unique only across successful releases of a given fleet, hence you can get an AmbiguousReleaseError in case i) your first `git push` failed, or you are `git push`ing the same commit to more than one fleets.)

The same SDK method already supports using id or commit.
You can check how it's done for devices here:
See: https://github.com/balena-io/balena-cli/blob/88fc3f77146144eb56b906aa413ef2f8e6b84c95/lib/commands/device/index.ts#L68

Reply:
Does this mean that you can have a failed push and a successful push with same commit? I thought pushing a release from the same git commit would just update the release, still making it unique.

I'm not sure about useful is adding support for release ids here, as the only place the release id can be obtained is from the builder logs and from the API

Reply:
I'm not sure whether that has changed recently, but for sure when I joined each git push created a new release and the rule that allows that hasn't been changed to anything stricter yet:
See: Rule: It is necessary that each application that owns a release1 that has a status that is equal to "success" and has a commit1, owns at most one release2 that has a status that is equal to "success" and has a commit2 that is equal to the commit1.